packages/cli-e2e/verdaccio/storage/data/.verdaccio-db.json

```diff
@@ -0,0 +1 @@
+{"list":["@coveo/search-token-server","@coveo/vue-cli-plugin-typescript","@coveo/search-token-lambda","@coveo/cra-template","@coveo/create-atomic","@coveo/angular"],"secret":"404d80afa5b41a75055bc4ac54ddfb3c41228a967ea5a2a8f3134e137da92691"}
```

Comment:
What's that secret ?

Reply:
It's a secret generated and used by verdaccio internally, I cleaned it up tho, as it is a generated file, it ain't useful.